fideslog/api/database/registrations.py

```diff
@@ -0,0 +1,91 @@
+from datetime import datetime, timezone
+from logging import getLogger
+from typing import List, Optional
+
+from sqlalchemy.exc import NoResultFound
+from sqlalchemy.orm import Session
+
+from ..models.models import Registration as RegistrationORM
+from ..schemas.registration import Registration
+
+log = getLogger(__name__)
+
+
+def get(
+    database: Session,
+    count: Optional[int],
+    offset: Optional[int],
+) -> List[RegistrationORM]:
+    """
+    Return existing registrations.
+    """
+
+    log.debug("Fetching registrations")
+    if count is not None:
+        return database.query(RegistrationORM).limit(count).offset(offset or 0).all()
```

Comment:
Consider defaulting `offset: int = 0` instead of setting manually here

Reply:
🤦‍♂️